libbeat/scripts/Makefile

```diff
@@ -219,11 +219,11 @@ update: python-env
 
```

Comment:
hi @ruflin i think i call the “”python-env” before ”update：”
and it should have been actived already